unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/UsersAroundListHUD/UsersAroundListHUDListElementView.cs

```diff
@@ -58,6 +59,21 @@ public void SetUserProfile(UserProfile profile)
         {
             profile.OnFaceSnapshotReadyEvent += SetAvatarPreviewImage;
         }
+
+        if (FriendsController.i)
+        {
+            if (FriendsController.i.friends.TryGetValue(profile.userId, out FriendsController.UserStatus status))
+            {
+                SetupFriendship(status.friendshipStatus);
+            }
+            else
+            {
+                SetupFriendship(FriendshipStatus.NONE);
+            }
+
+            FriendsController.i.OnUpdateFriendship -= OnFriendActionUpdate;
+            FriendsController.i.OnUpdateFriendship += OnFriendActionUpdate;
+        }
```

Comment:
Let's invert this to avoid indendation! I would suggest to make the comparison to null explicit as well, some types might have implicit operators to bool and it's a good practice to avoid them.

Reply:
I don't like inverting if inside methods that do other stuff also so I created a new one for friend's setup and inverted it there :P